flepimop/gempyor_pkg/src/gempyor/utils.py

```diff
@@ -544,6 +655,19 @@ def create_resume_file_names_map(
     parquet file types and environmental conditions. The function adjusts the file name mappings
     based on the operational block index and the location of the last job output.
 
+    Args:
+        resume_discard_seeding:
+        flepi_block_index:
+        resume_run_index:
+        flepi_prefix:
+        flepi_slot_index:
+        flepi_run_index:
+        last_job_output:
+
+    Return:
+        Dict[str, str]: A dictionary where keys are input file paths and values are corresponding
```

Comment:
The second part of this sentence, the "The paths may be modified...." might be better off going in `Notes:` or in a description under the title instead.